### PR TITLE
Dread: Add Burenia Crossbomb trick to database

### PR DESCRIPTION
**Pull request by Logarhythmic**
Added: Cross bomb movement to the database for the missile tank in Burenia: Main Hub Tower Top.
Added: 2 videos to the database for cross bomb movment in Burenia.

### CODE REVIEW

**Comment by Miepee** [comment]
For the future, please make the PR slightly more readable :)

File: randovania/games/dread/json_data/Burenia.txt [hunk 1]
```diff
@@ -1036,6 +1038,7 @@ Extra - asset_id: collision_camera_010
               All of the following:
                   Lay Cross Bomb
                   Any of the following:
+                      # Movement (Intermediate): https://youtu.be/Dy24YPtX0A8
```
Comment (Tyranisaur) [inline randovania/games/dread/json_data/Burenia.txt:1041]: The trick that was in there before was meant to be a simple sideways jump with cross bomb midair. So I would prefer if this kind of launch was added as a separate trick.

**Review by Tyranisaur — CHANGES_REQUESTED** [review]

**Review by XenoWars — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,14 +26,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Added: 3 videos to the logic database for a diagonal bomb jump in Ghavoran, a single-wall jump in Cataris, and a diffusion abuse trick in Artaria.
+- Added: 2 videos to the logic database for cross bomb movement in Burenia.
+- Added: Cross Bomb Launch (CBL) is now its own trick in the database. 
 ##### Artaria
 
 - Changed: EMMI Zone Spinner: The connection to the pickup that is available before flipping the spinner now also requires door lock rando and Highly Dangerous Logic to be enabled.
 
 ##### Burenia
 
+- Added: The missile tank in Burenia Main Hub Tower Top can be gotten with morph, cross bombs, and movement (intermediate).
```
Comment (XenoWars) [inline CHANGELOG.md:38]: I just approved, but i forgot about this. Should be Cross Bomb Launch instead of Movement.

**Review by XenoWars — CHANGES_REQUESTED** [review]

File: randovania/games/dread/json_data/header.json [hunk 3]
```diff
@@ -1200,6 +1200,11 @@
                 "long_name": "Ledge Warp",
                 "description": "A frame perfect trick that allows you to warp to a ledge you have previously been at by entering Morph Ball during the mantle animation. See https://youtu.be/3QE0tEC2tgc",
                 "extra": {}
+            },
+            "CBL": {
+                "long_name": "Cross Bomb Launch",
+                "description": "Sliding and morphing into an exploding cross bomb gives Samus a large amount of horizontal velocity.",
```
Comment (Tyranisaur) [inline randovania/games/dread/json_data/header.json:1206]: I think I would have phrased this as "high horizontal velocity". I don't know why, but some reason, taking about an amount of velocity seems weird to me. Might just be a high level of pedantry at this point.

**Review by Tyranisaur — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Logarhythmic) [reply CHANGELOG.md:38]: Changed.

File: randovania/games/dread/json_data/header.json (hunk 3 above)
Comment (Logarhythmic) [reply randovania/games/dread/json_data/header.json:1206]: Updated to "high horizontal velocity". From a mathy perspective, one can add or subtract varying amounts of velocity. I've not strong preference on the wording.

**Comment by ThanatosGit** [comment]
With a new version released comes the changelog conflicts

Could you please fix that?

**Pull request closed without merging**